src/DataClass/Partnership.php

```diff
@@ -0,0 +1,270 @@
+<?php
+
+namespace App\DataClass;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Data Class for PartnerType
+ */
+class Partnership
+{
+    /**
+     * @Assert\NotBlank()
+     * @Assert\Length(min="2", max="255")
+     */
+    private string $businessName;
+
+    /**
+     * @Assert\NotBlank()
+     */
+    private string $expertise;
```

Comment:
Your expertise seems to be one value in a list of given choice. You could validate that it is one of this choices.

Reply:
https://symfony.com/doc/current/reference/constraints/Choice.html would be better than regex in that case